pysages/backends/core.py

```diff
@@ -42,18 +43,62 @@ def set_backend(name):
 
 
 def bind(context, sampling_method, **kwargs):
-    """Couples the sampling method to the simulation."""
-    #
+    """Couples the sampling method to the simulation.
+
+    context -- hoomd simulation context
+    sampling_method -- pysages sampling_method
+    callback -- callback method with call signature `callback(snapshot, state)`
+      called after PySages updated. Example: logging of CVs.
+    """
```

Comment:
```suggestion
    context -- Backend simulation context
    sampling_method -- PySAGES sampling method
    callback -- callback method with call signature `callback(snapshot, state, timestep)`
      called after each pysages update. Example: logging of CVs.
    """
```

Unrelated, but this makes me think that for easy discoverability we should probably add a couple of methods to list available sampling methods and collective variables. We could open an issue to track this.

I think that I would add callback as an explicit argument here:

```python
def bind(context, sampling_method, callback: Callable = None, **kwargs):
   # ...
```